go/vt/sqlparser/ast.go

```diff
@@ -741,22 +741,31 @@ type DDL struct {
 
 	// VindexCols is set for AddColVindexStr.
 	VindexCols []ColIdent
+
+	// VschemaUpdates are key=value expressions to set vschema values like `Authoritative`
+	VschemaUpdates UpdateExprs
+
+	// AuthColumn is set for add/drop Authoritative Columns
+	AuthColumn *ColumnDefinition
```

Comment:
Looks like today vitess doesn't need to know how to parse column information from alter ddls.

A question to consider: when adding support for parsing some detail from a vschema alter statement, is it worthwhile to try to align the AST with the direction it would go in if trying to support parsing mysql schema alter statements?

Here's the alter AST in the mysql 8 code, for reference: https://github.com/mysql/mysql-server/blob/8e797a5d6eb3a87f16498edcb7261a75897babae/sql/sql_alter.h#L168.